### PR TITLE
Changed error handling for toxicity

### PR DESCRIPTION
**Pull request by jindaln**

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/NewPostServlet.java [hunk 1]
```diff
@@ -64,7 +64,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             commentId = DatastoreManager.insertCommentInDatastoreIfNonToxic(name, comment);
         } catch(ToxicCommentException e) {
             logger.error(e);
-            throw new ServletException("Error: " + e.getMessage(), e);
+            String redirect = (feedBool == true) ? "feed.html?name=" + URLEncoder.encode(repName) :
```
Comment (v1shanker) [inline src/main/java/com/google/sps/servlets/NewPostServlet.java:67]: feedBool is sufficient

File: src/main/java/com/google/sps/servlets/RepUploadImageServlet.java [hunk 2]
```diff
@@ -74,7 +74,7 @@ private String getBlobKeyUrl(HttpServletRequest request, String formInputElement
         if (blobKeys == null || blobKeys.isEmpty()) {
             return(Constants.DEFAULT_POLITICIAN_IMAGE_URL);
         } else {
-            return (SERVE_URL + blobKeys.get(0).getKeyString());
+            return(SERVE_URL + blobKeys.get(0).getKeyString());
```
Comment (v1shanker) [inline src/main/java/com/google/sps/servlets/RepUploadImageServlet.java:77]: space after return

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -687,11 +702,12 @@ async function submitRepQuestionnaire() {
         }
     }
     //Remove * on the end of the last platform string if it exists
-    var lastPlatform = listOfPlatforms[listOfPlatforms.length - 1];
-    lastPlatform = (lastPlatform[lastPlatform.length - 1] == "*") ? 
+    if (listOfPlatforms != []) {
```
Comment (v1shanker) [inline src/main/webapp/script.js:705]: is there an isEmpty or something like that?

Comment (jindaln) [reply src/main/webapp/script.js:705]: or rather isNotEmpty?

**Pull request merged**